### PR TITLE
Update intro on backup

### PR DESCRIPTION
**Pull request by theabmitra**

### CODE REVIEW

File: content/en/docs/uninstall/backup/_index.md [hunk 1]
```diff
@@ -8,4 +8,4 @@ draft: false
 The Verrazzano platform comprises a comprehensive set of open source components, linked together to provide a modern, reliable, and secure platform to deploy cloud native applications.
 These documents will help you back up and restore data and configurations in Keycloak, OpenSearch, and Rancher.
 
-The backup procedures are independent and you can invoke them in the order of your choice. For most of the backup functionality, these procedures rely on an OCI object store as the back end.
+The backup procedures are independent and you can invoke them in the order of your choice. For most of the backup functionality, these procedures rely on an object store as the back end, such as Oracle Cloud Infrastructure Object Store.
```
Comment (rosemarymarano) [inline content/en/docs/uninstall/backup/_index.md:11]: -> such as the Oracle Cloud Infrastructure (OCI) object store.   **OR**
-> such as the OCI object store. 

(Is there some reason why you capitalized the words Object Store?)

**Review by rosemarymarano — COMMENTED** [review]
One comment, please address.

File: content/en/docs/uninstall/backup/_index.md (hunk 1 above)
Comment (theabmitra) [reply content/en/docs/uninstall/backup/_index.md:11]: nothing specific , this was a comment i recieved where they wanted me to word it that way. ill change that

File: content/en/docs/uninstall/backup/_index.md [hunk 2]
```diff
@@ -8,4 +8,4 @@ draft: false
 The Verrazzano platform comprises a comprehensive set of open source components, linked together to provide a modern, reliable, and secure platform to deploy cloud native applications.
 These documents will help you back up and restore data and configurations in Keycloak, OpenSearch, and Rancher.
 
-The backup procedures are independent and you can invoke them in the order of your choice. For most of the backup functionality, these procedures rely on an object store as the back end, such as Oracle Cloud Infrastructure Object Store.
+The backup procedures are independent and you can invoke them in the order of your choice. For most of the backup functionality, these procedures rely on an object store as the back end, such as OCI object Store.
```
Comment (rosemarymarano) [inline content/en/docs/uninstall/backup/_index.md:11]: -> store

**Review by rosemarymarano — COMMENTED** [review]
One more fix

**Pull request merged**